sw/host/opentitanlib/src/util/num_de.rs

```diff
@@ -208,6 +208,19 @@ impl_parse_int_enc!(OctEncoded, 8, fmt::Octal, "0o");
 impl_parse_int_enc!(DecEncoded, 10, fmt::Display, "");
 impl_parse_int_enc!(HexEncoded, 16, fmt::LowerHex, "0x");
 
+impl ParseInt for Vec<u8> {
+    type FromStrRadixErr = ParseIntError;
+
+    fn from_str_radix(src: &str, radix: u32) -> Result<Self, ParseIntError> {
+        let mut bytes = vec![];
+        for digit_bytes in src.as_bytes().rchunks(2) {
```

Comment:
Can `src`'s length be odd? What would `.rchunks(2)` do in that case? Is the possible to drop the MSB?

Would it make sense to use `sw/host/opentitanlib/src/util/bigint.rs` (by defining a big enough type) or `num_bigint_dig::BigUint` here?

Reply:
Honestly, this was moved from `sw/host/opentitanlib/src/otp/otp.rs` to keep manifest deserialization intact. This shouldn't be needed once the manifest is moved to serde-annotate. 

For odd length inputs the last chunk will be only length 1 https://docs.rs/bitvec/latest/bitvec/slice/struct.RChunks.html.

Reply:
SG